Fix Grafana & Kiali for Service Catalog

mszostok opened the pull request:
Description:
- Fix Grafana & Kiali for Service Catalog
  - Also query for showing the OSB API request was fixed, now it shows req/min

<img width="1852" alt="image" src="https://user-images.githubusercontent.com/17568639/83802189-9a346e80-a6aa-11ea-96c2-dddb19e87e43.png">


Fixed Kiali: 
- https://kiali.schemas.kymcia.online/console/namespaces/kyma-system/services/service-catalog-catalog-webhook?duration=60
- https://kiali.schemas.kymcia.online/console/namespaces/kyma-system/services/service-catalog-catalog-controller-manager?duration=60

Fixed Grafana:
- https://grafana.schemas.kymcia.online/d/t9EmddWWzn/kyma-service-catalog-fixed

polskikiel commented:
I have 3 things: 
![image](https://user-images.githubusercontent.com/21159554/83881564-2a24f780-a741-11ea-82fa-a9ee94c12c04.png)
When you display nulls as zeros it looks better imo

There is also some inconsistency between memory and CPU graphs, for current memory usage you use prefix `Current:` and for CPU you don't:
![image](https://user-images.githubusercontent.com/21159554/83883850-58f09d00-a744-11ea-92f8-300cbd1aa482.png)

On the above ss you can see that we also have now also a field: `Current: container_name`. If you want to get rid of it, add `,container!=""` to query filters.

mszostok commented:
Hi @polskikiel 
Thanks for the review, I've adjusted the CPU and Memory charts and also added the `,container!=""` arg to Prometheus query.

About nulls as zeros IMO is better to do not apply too much manipulation on our charts, if you see that chart is empty for some time that you know that there can be some problem pods restarts, wrong servicemonitor modification, etc. Without that, you will always see zero even if something will be broken

mszostok commented:
Issue for the Helm Broker: https://github.com/kyma-project/helm-broker/issues/101
